src/nodedef.cpp

```diff
@@ -33,6 +33,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "exceptions.h"
 #include "debug.h"
 #include "gamedef.h"
+#include <fstream> // For applyTextureOverrides()
```

Comment:
Usually if you comment an `include`, if you have a "for $ABC", then ABC holds the function names you use, not the function you use the included functions in.